tests/phpunit/tests/media.php

```diff
@@ -3777,6 +3785,12 @@ function( $attr ) {
 				return $attr;
 			}
 		);
+		add_filter(
+			'wp_omit_loading_attr_threshold',
+			function() {
+				return 1;
+			}
```

Comment:
As code snippet is repeated 3 times in this line. How about a util method on the class, to return 1.

Reply:
Either that, or to avoid even more duplicate code, how about we create a method like:
```php
	public function force_omit_loading_attr_threshold( $threshold ) {
		add_filter(
			'wp_omit_loading_attr_threshold',
			static function() use ( $threshold ) {
				return $threshold;
			}
		);
	}
```

Reply:
@spacedmonkey @felixarntz I have gone ahead and created a util method like you both suggested.